### PR TITLE
[2021.2] Bump SRP packages to 12.1.0

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

Backport of #5812.
- Bumped SRP packages to 13.1.0
- Moved some release notes to the new section, as they were added after the commit used for the current vendoring (and therefore correspond to the next vendoring).

---
### Testing status
Pack & Test all packages (upm-ci validation): https://unity-ci.cds.internal.unity3d.com/project/902/branch/2021.2%2Fsrp%2Fbump-packages/jobDefinition/.yamato%2F_projectcontext.yml%23all_package_ci_project_2021.2

### CODE REVIEW

File: com.unity.render-pipelines.core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [12.1.0] - 2021-09-23
+
```
[the remark this answers is not in the document]
Comment (sebastienlagarde) [reply com.unity.render-pipelines.core/CHANGELOG.md:8]: Version Updated
The version number for this package has increased due to a version update of a related graphics package.

File: com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [12.3.0] - 2021-09-15
+
```
Comment (sebastienlagarde) [inline com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md:8]: need to say that you bumped to SRP 12.1.0 version

File: com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,13 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [12.2.0] - 2021-09-24
+## [12.3.0] - 2021-09-24
```
Comment (sebastienlagarde) [inline com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md:7]: you shouldn chant' the revision where the fix landed (even if I am agree we haven't promote this verison officially). Move ## [12.2.0] - 2021-09-24 up

**Review by sebastienlagarde — COMMENTED** [review]
there is also documentation.cs to bump and version.hlsl.  (yes bumping is a nightmare :) )

**Comment by theo-pnv** [comment]
Applied requested changes @sebastienlagarde, thanks.
Removed all modifications made to the templates (except the hdrp or urp version they're using), as you are already bumping them in other PRs https://github.com/Unity-Technologies/Graphics/pull/5799.

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**